src/cache/s3.rs

```diff
@@ -67,7 +64,7 @@ impl S3Cache {
         let provider =
             AutoRefreshingProvider::new(ChainProvider::with_profile_provider(profile_provider))?;
         let bucket_name = bucket.to_owned();
-        let bucket = std::sync::Arc::new(Bucket {
+        let _bucket = std::sync::Arc::new(Bucket {
```

Comment:
This is obviously either dead code that should be removed, or made use of. So the warning should either stay or proper fix be applied.

Reply:
remove unused rusoto_s3::Bucket, as it carries only bucket_name that is stored later in S3Cache object anyway.

Done in : 145876b